utils/examples/generate-examples.py

```diff
@@ -0,0 +1,108 @@
+import argparse
+import os
+import sys
+import pprint
+import logging
+
+import verovio
+
+IMAGES_PATH = "../../build/assets/images/GeneratedImages"
+
+VRV_OPTIONS: dict = {
+   'scale': 10,
+   'pageWidth': 1500,
+   'adjustPageHeight': True,
+   'adjustPageWidth': True,
+   'footer': 'none',
+   'header':'none',
+   'openControlEvents': True,
+   'outputFormatRaw': True,
+   'removeIds': True,
+   'svgFormatRaw': True,
+   'svgHtml5': True,
+   'svgRemoveXlink': True,
+   'svgViewBox': True,
+    'xmlIdSeed': 1
+}
+
+success: bool = True
+
+if __name__ == "__main__":
+    description = """
+        Renders SVG for the MEI examples. Will generate any new examples found; to
+        re-generate all examples use the '--clean' option.
+    """
+    parser = argparse.ArgumentParser(description=description)
+    parser.add_argument("--clean", action="store_true", help="Re-generate all examples.")
+    verbose_group = parser.add_mutually_exclusive_group()
+    verbose_group.add_argument("--verbose", "-v", action="store_true")
+    verbose_group.add_argument("--debug", "-d", action="store_true")
+
+    args = parser.parse_args()
+    if args.debug:
+        level_msg: str = "Debug"
+        level = logging.DEBUG
+    elif args.verbose:
+        level_msg: str = "Info"
+        level = logging.INFO
+    else:
+        level_msg: str = "Error"
+        level = logging.ERROR
+
+    logging.basicConfig(format="[%(asctime)s] [%(levelname)8s] %(message)s (%(filename)s:%(lineno)s)",
+                        level=level)
+    log = logging.getLogger(__name__)
+    log.info("Running at logging level %s", level_msg)
+
+    verovio.enableLog(verovio.LOG_DEBUG)
+    tk = verovio.toolkit()
+    # version of the toolkit
+    log.info("Using Verovio %s", tk.getVersion())
+
+    # keep all the options to be able to reset them for each example
+    default_options: dict = tk.getDefaultOptions()
+    # Overwrite the default options with our locally-defined options
+    default_options.update(VRV_OPTIONS)
+
+    for file in os.listdir(IMAGES_PATH):
+        # Skip everything that is not an .mei or .xml file
+        if not(file.endswith(".mei")) and not(file.endswith(".xml")): 
+            continue
+        
+        options: dict = default_options.copy()
```

Comment:
Not sure why you need to make a copy of the options dict on every pass, or reset the options on the toolkit. It should be enough to set `tk.setOptions` once.

Reply:
This remains from when we had (potentially) specific options read from the example. It can now be moved up and set only once.